tools/update-db.sh

```diff
@@ -14,18 +15,23 @@ source /etc/makepkg.conf
 cd "$ROOT"/out
 curl -fOs "https://arch.osamc.de/proaudio/${CARCH}/proaudio.{db,files}.tar.gz"
```

Comment:
Hier Fehler zu unterdrücken halte ich für falsch.

Reply:
Then you have to check for the existence of the db file afterwards before going on.

Reply:
> Then you have to check for the existence of the db file afterwards before going on.

This is also now covered by the extraction step. If the file doesn't exist, the whole step fails with an error.